src/CommunityVoices/App/Website/Bootstrap/Router.php

```diff
@@ -47,4 +47,12 @@ public function getRoutes()
     {
         return $this->routes;
     }
+
+    public function isLocal()
+    {
+      if((getenv('SERVER') === 'environmentaldashboard.org')) {
```

Comment:
Why can't we just return `!(getenv('SERVER' === 'environmentaldashboard.org'))`?

Reply:
I feel like there is absolutely no need for boolean constants here.

Reply:
Resolving since this code no longer exists.